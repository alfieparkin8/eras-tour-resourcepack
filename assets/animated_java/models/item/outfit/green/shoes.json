{
	"parent": "animated_java:item/outfit/shoes",
	"textures": {
		"2": "animated_java:item/outfit/green_layer_1",
		"4": "animated_java:item/outfit/green_layer_1",
		"6": "animated_java:item/outfit/green_layer_2"
	}
}