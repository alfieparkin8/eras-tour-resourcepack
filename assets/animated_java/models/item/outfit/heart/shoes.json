{"parent":"animated_java:item/outfit/shoes","textures":{"2":"minecraft:block/outfits/lover/heart_layer_1"}}