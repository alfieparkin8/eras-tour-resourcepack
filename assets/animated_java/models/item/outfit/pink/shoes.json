{"parent":"animated_java:item/outfit/shoes","textures":{"2":"minecraft:block/outfits/1989/pink_layer_1","4":"minecraft:block/outfits/1989/pink_layer_1","6":"minecraft:block/outfits/1989/pink_layer_2"}}