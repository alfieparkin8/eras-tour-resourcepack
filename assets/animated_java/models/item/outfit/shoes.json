{
	"textures": {
		"4": "minecraft:block/outfits/lover/sunset_boots_layer_1"
	},
	"elements": [
		{
			"from": [
				5.25,
				-7.75,
				6.25
			],
			"to": [
				10.75,
				5.75,
				11.75
			],
			"faces": {
				"north": {
					"uv": [
						2,
						10,
						1,
						16
					],
					"texture": "#4"
				},
				"east": {
					"uv": [
						2,
						10,
						3,
						16
					],
					"texture": "#4"
				},
				"south": {
					"uv": [
						4,
						10,
						3,
						16
					],
					"texture": "#4"
				},
				"west": {
					"uv": [
						1,
						10,
						0,
						16
					],
					"texture": "#4"
				},
				"up": {
					"uv": [
						2,
						8,
						1,
						10
					],
					"texture": "#4"
				},
				"down": {
					"uv": [
						3,
						8,
						2,
						10
					],
					"texture": "#4"
				}
			}
		},
		{
			"from": [
				9.25,
				-7.75,
				6.25
			],
			"to": [
				14.75,
				5.75,
				11.75
			],
			"faces": {
				"north": {
					"uv": [
						2,
						10,
						1,
						16
					],
					"texture": "#4"
				},
				"east": {
					"uv": [
						2,
						10,
						3,
						16
					],
					"texture": "#4"
				},
				"south": {
					"uv": [
						4,
						10,
						3,
						16
					],
					"texture": "#4"
				},
				"west": {
					"uv": [
						1,
						10,
						0,
						16
					],
					"texture": "#4"
				},
				"up": {
					"uv": [
						2,
						8,
						1,
						10
					],
					"texture": "#4"
				},
				"down": {
					"uv": [
						3,
						8,
						2,
						10
					],
					"texture": "#4"
				}
			}
		}
	],
	"display": {
		"head": {
			"rotation": [
				0,
				180,
				0
			]
		}
	}
}