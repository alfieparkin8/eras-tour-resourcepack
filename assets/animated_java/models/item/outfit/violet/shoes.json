{"parent":"animated_java:item/outfit/shoes","textures":{"2":"minecraft:block/outfits/lover/violet_layer_1"}}