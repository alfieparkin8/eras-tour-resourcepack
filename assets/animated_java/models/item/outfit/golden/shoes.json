{"parent":"animated_java:item/outfit/shoes","textures":{"0":"minecraft:block/outfits/lover/golden_layer_1","1":"minecraft:block/outfits/lover/heart_layer_1","2":"minecraft:block/outfits/lover/golden_layer_1","3":"minecraft:block/outfits/lover/violet_layer_1","4":"minecraft:block/outfits/lover/golden_boots_layer_1","5":"minecraft:block/outfits/lover/golden_boots_layer_1"}}